src/fastly/include-util.js

```diff
@@ -13,6 +13,14 @@
 const fs = require('fs-extra');
 const path = require('path');
 
+function regex(content, filePath) {
+  return content.replace(/\{"regex:block.rgx"\}/g, () => {
+    const name = path.resolve(filePath);
+    const c = fs.readFileSync(name).toString().split('\n').join('');
```

Comment:
```suggestion
    const c = fs.readFileSync(name).toString().split('\n').join('|');
```

That way we can skip the trailing pipe in the regex files.

Reply:
```suggestion
    const c = fs.readFileSync(name)
      .toString()
      .split('\n')
      .map(l => l.replace(/[ ]+#.*$/, '')) // enables comments at the end of the line
      .join('|');
```

even better: comments that can explain each line